app/assets/stylesheets/bootstrap_overrides.scss

```diff
@@ -4,15 +4,19 @@ body {
 	padding-right: 200px;
 }
 .navbar-brand {
-	font-size: 30px;
+	font-size: 40px;
 	font-weight: bold;
 } 
-.the {
+.cyan {
 	color: #00e5e6;
 }
-.petite {
+.black {
 	color: #000000;
 }
-.life {
+.pink {
```

Comment:
I'm glad you made this change! A couple more tweaks could be:
1. Since these are specifically text colors, maybe name the classes `.cyan-text`, `.black-text`, etc.
2. Since this is SCSS, let's use variables! That way you can easily reference these colors throughout your styles by their variable names (instead of gross hex values... gross). Something like (at the top of the file, or in a separate partial file called `_colors.scss`):
```
$cyan: #00e5e6;
$black: #000000;
$pink: #ff00ff;
```
Then, in your code, you can just have stuff like:
```
.cyan-text { color: $cyan }
.black-text { color: $black }
.pink-text { color: $pink }
```
:yum: